ui/job-view/pushes/JobGroup.jsx

```diff
@@ -146,6 +146,20 @@ export class JobGroupComponent extends React.Component {
     const { expanded } = this.state;
     const { buttons, counts } = this.groupButtonsAndCounts(groupJobs, expanded);
 
+    const successfulJobTypeNames = new Set();
+    for (const job of groupJobs) {
+      if (job.result === 'success') {
+        successfulJobTypeNames.add(job.job_type_name);
```

Comment:
This can be a pretty big list (>10k items) but I have not observed a negative impact.

Reply:
should I look into other solution?
(we could build a set of failed jobs first, and filter it with successful jobs to get a set of intermittent jobs)
or can we go with this?